app/controllers/concerns/respondable.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Respondable
+  extend ActiveSupport::Concern
+
+  def deleted_successfully_render(record)
+    render json: { message: "#{record.class} deleted successfully." }, status: :ok
```

Comment:
`status: :no_content` ?

Reply:
If I use :no_content status, the response body will be empty, so we can't render a successfully message. Do we follow using ok or no_content and the response body doesn't having a message?

Reply:
up to you